frontend/src/APIClients/mutations/AuthMutations.ts

```diff
@@ -0,0 +1,57 @@
+import { gql } from "@apollo/client";
+
+export const LOGIN = gql`
+  mutation Login($email: String!, $password: String!) {
```

Comment:
why is `login` a mutation instead of a query? are we changing stuff from a backend perspective?